src/tech/yk_structure.md

```diff
@@ -1,36 +1,38 @@
 # The Structure of the JIT Runtime
 
 The `yk` repo's structure (and build system) is somewhat unusual as the code is
-spread over two workspaces. This is because we need to separately compile parts
-of the codebase with different configurations.
+spread over different workspaces. This is because we need to separately compile
+parts of the codebase with different configurations.
 
 Code that we expect to be able to hardware trace needs to be compiled with `-C
 tracer=hw`. Crucially, since hardware tracing relies on blocks not being
 reordered by LLVM, this flag disables optimisations. However, the JIT runtime
 will never be traced and therefore we can (and should) optimise this code.
 Further we want to be able to optimise the dependencies of the JIT runtime.
 
-To this end, we have two Rust workspaces in the `yk` repo: the "internal"
-(optimised) workspace, and the "external" (unoptimised) workspace. The external
-workspace then talks to the internal workspace via `extern` functions defined
-in the `ykshim` crate.
+To this end, we have three Rust workspaces in the `yk` repo:
+ - The top-level xtask workspace: our build system.
+ - The untraced workspace: the "always optimised" JIT runtime.
```

Comment:
Should we put `untraced` and `traced` in backticks here (and elsewhere in this document)?

Reply:
Like so: 59103b5

Reply:
Excellent!